### DISCUSSION TITLE
Disabling cache for forms needed?

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#16382] [Q&A / Support] [answered]
Hello there,
i build an areabrick which outputs a symfony form for the user. The form uses CSRF Tokens and as they need to be different for every user, my question is, do i need to disable cache (and/or full page cache) for documents that include that areabrick?

cheers,
george

**Comment by dpfaffenbauer** [comment, marked as answer]
no, Pimcore sees that a Session is being used and does not store the page into the full-page cache nor trying to load it from the full-page cache since there is a valid session

**Reply by ghost** [reply]
Thank you very much for the quick answer! One subsequent question: does this work for cookies set via setcookie() as well or only session cookies?

**Reply by dpfaffenbauer** [reply]
AFAIK: Symfony Session only.

**Reply by ghost** [reply]
thx!
